modules/gitops-release-notes-1-3-0.adoc

```diff
@@ -0,0 +1,139 @@
+----
+$ oc adm taint nodes -l node-role.kubernetes.io/infra
+infra=reserved:NoSchedule infra=reserved:NoExecute
+----
+. Add `runOnInfra` toggle in the `GitOpsService` Custom Resource. Run the following command:
++
+[source,yaml]
+----
+apiVersion: pipelines.openshift.io/v1alpha1
+kind: GitopsService
+metadata:
+  name: cluster
+spec:
+  runOnInfra: true
+----
+. Add `tolerations` to the `GitOpsService` Custom Resource if taints have been added to the nodes, for example:
++
+[source,yaml]
+----
+  spec:
+    runOnInfra: true
+    tolerations:
+    - effect: NoSchedule
+      key: infra
+      value: reserved
+    - effect: NoExecute
+      key: infra
+      value: reserved
+----
+. Verify that the workloads in the `openshift-gitops` namespace are now scheduled on infrastructure nodes by viewing `Pods -> Pod details` for any pod in the console UI.
+
+[NOTE]
+====
+Any manually added `nodeSelectors` and `tolerations` added to the default Argo CD Custom Resource will be overwritten by toggle and `tolerations` in the `GitOpsService Custom Resource`.
+====
+* Internal communications in Argo CD can now be secured using `TLS` and OpenShift cluster certificates. Argo CD routes can now leverage OpenShift cluster certificates in addition to using external certificate managers such as cert-manager.
+* Gain insights into the GitOps environments directy from the DevConsole using the improved environment details view in DevConsole for OCP 4.9.
+* You will now have access to custom health checks in Argo CD for `DeploymentConfig` resources, Route resources, and `OLM` operators.
+* The GitOps Operator will now conform the naming conventions recommended by the latest operator-sdk by introducing the following changes:
+** The prefix `gitops-operator-` will be added to all resources
+** Service account will be renamed to `gitops-operator-controller-manager`
+
+
+[id="fixed-issues-1-3_{context}"]
+== Fixed issues
+The following issues were resolved in the current release:
+
+* Previously, if you were setting a new namespace to be managed by a new instance of Argo CD, it would immediately be “Out Of Sync” due to the new roles and bindings that the operator creates to manage that new namespace. This behavior is fixed.  link:https://issues.redhat.com/browse/GITOPS-1384[GITOPS-1384]
+
+[id="known-issues-1-3_{context}"]
+== Known issues
+These are the known issues in {gitops-title} 1.3.0:
+
+* While migrating from Dev to Keycloak, you may experience login issues with Keycloak. link:https://issues.redhat.com/browse/GITOPS-1450[GITOPS-1450]
+
+Prevention:
+
+* When migrating, uninstall Dex by removing the `.spec.dex` section found in the Argo CD custom resource. Allow a few minutes for Dex to uninstall completely, and then proceed to install Keycloak by adding `.spec.sso.provider: keycloak` to the Argo CD custom resource.
+
+Workarounds:
+
+* If you run into the above issue, uninstall Keycloak by removing `.spec.sso.provider: keycloak` and then re-install.
```

Comment:
When a user skims the KIs the separate sections give the impression that there are three KIs but actually there is only one. Suggest making the prevention and workaround as paragraphs. Also there is only one workaround so Workarounds does not sound right.
Suggest:
As a prevention, when migrating, uninstall Dex by removing....
As a workaround, If you run into the above issue, uninstall...